tests/testcases/030_check-network.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: kube_control_plane[0]
   vars:
-    test_image_repo: busybox
+    test_image_repo: k8s.gcr.io/busybox
```

Comment:
How about specifying `{{ kube_image_repo }}/` instead of `k8s.gcr.io/`?
The fixed string(`k8s.gcr.io/`) affects offline deployment, I guess.

Reply:
We can't, those vars are not injected in tests playbooks :(

Reply:
ah, I see.
Then it is fine for me to merge current one.